_posts/2020-03-11-sr2020-covid-19-updates.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: news
+title: COVID-19 Coronavirus Student Robotics Updates
+---
+
+As you will be aware, the COVID-19 situation is quickly evolving, having recently been delcared a pandemic. 
+Many large events are being cancelled and we want to be public about how we are responding to the latest news.
+
+The Competition Team are working closely with the Student Robotics trustees so that we can make the most sensible decision.
+
+## The Competition
+
+At the time of writing, we are carefully considering whether the competition can go ahead. We will be announcing our decision by Wednesday 18th March at the very latest. 
```

Comment:
The prevailing message here should be that we want this competition to run, we're trying everything we can to make it happen. I would move away from language like "considering whether it can go ahead", and lean more into whether we can figure out a way to run it safely.